src/components/AnonymousChat/child/ChannelImage.tsx

```diff
@@ -130,7 +131,7 @@ const ChannelImage = ({
         />
       );
     }
-    if (!isAnonymousChannel && isAnonymousOldPostMaker)
+    if (!isAnonymousChannel && isAnonymousOldPostMaker) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding curly braces to the `if` statement improves readability and consistency with other parts of the code that use curly braces for block statements. This is a good practice to follow.


<!-- This is an auto-generated comment by OSS CodeRabbit -->